mat-server-commons/src/main/java/mat/model/Author.java

```diff
@@ -0,0 +1,88 @@
+package mat.model;
+
+import com.google.gwt.user.client.rpc.IsSerializable;
+
+import javax.persistence.Column;
+import javax.persistence.Id;
+import javax.persistence.Transient;
+import java.util.Objects;
+
+//@Entity
```

Comment:
Is this optional?

Reply:
Um not using it as a Entity object, This is a cut n past from MAT.

This is being used for XML marshaling only to convert data.